corehq/apps/reports_core/filters.py

```diff
@@ -55,9 +58,23 @@ def get_value(self, request_params, user=None):
             kwargs.update(
                 {param.name: request_params[param.name] for param in self.params if param.name in request_params}
             )
-            return self.value(**kwargs)
+            choice_list = self.value(**kwargs)
         else:
-            return self.default_value(**kwargs)
+            choice_list = self.default_value(**kwargs)
+        if (user
+                and domain
+                and self.is_location_filter()
+                and toggles.LOCATION_RESTRICTED_SCHEDULED_REPORTS.enabled(domain)):
```

Comment:
Is it necessary to do the feature flag check here?  It makes sense to feature flag the views, so only flagged domains have access to the feature, but it should be safe(r) to do the filtering any time a restricted user makes it to this point, regardless of whether the flag is checked.

Reply:
Code block is removed now